docs/internal/maintenance/SystemTesting.md

```diff
@@ -76,36 +76,36 @@ The set of configurations must include at least:
          The output from command should be "1".
 
    3. Install Zonemaster Engine
-      1. Install dependencies according to the [installation instruction](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Installation.md).
```

Comment:
“installation instruction” is better written as “installation instructions”.
Why not convert the links to the reference style, like we do in the rest of the documentation?

Reply:
Fixed